src/cpuinfo/SDL_cpuinfo.c

```diff
@@ -468,6 +468,8 @@ CPU_haveNEON(void)
     return 1;  /* ARMv8 always has non-optional NEON support. */
 #elif __VITA__
     return 1;
+#elif __3DS__
+    return 1;
```

Comment:
Is this true? I thought the 3DS only has VFPv2, not NEON.

Reply:
I'm struggling to find information about it, it may be just safer to turn it off.